src/tls/Makefile

```diff
@@ -341,3 +364,10 @@ test13:
 ifeq ($(OS),Windows_NT)
     EXTRA_PATH = PATH="/usr/x86_64-w64-mingw32/sys-root/mingw/bin/:$(PATH)"
 endif
+
+# FFI support - calling from C into miTLS
+#depends on tls-gen
+tls-ffi: libmitls.dll
+
+libmitls.dll: mitls.cmxa #tls-gen
```

Comment:
why is `tls-gen` commented out? this seems like a legit dependency for this target to have